templates/plugins-step.tmpl

```diff
@@ -0,0 +1,99 @@
+steps:
+{{- if .CosignPluginEnabled }}
+  - label: ":docker: get cosign key"
+    key: "getkey"
+    command: |
+      #!/bin/bash
+      echo "\$COSIGN_KEY_SECRET" > ${COSIGN_KEY_PATH}
+    plugins:
+{{- else }}
+  - command: ls
+    plugins:
+{{- end }}
+{{- if .SSMPluginEnabled }}
+      - ssh://git@git@github.com/equinixmetal/ssm-buildkite-plugin#{{ .CosignConfig.CosignVersion }}:
```

Comment:
1 too many `git@`s in the url

Reply:
Thanks for the review @oliver-equinix, the change has been addressed..